src/modules/rest/pipes/OptionalBoolPipe.ts

```diff
@@ -0,0 +1,8 @@
+import {PipeTransform, Injectable} from '@nestjs/common';
+
+@Injectable()
+export class OptionalBoolPipe implements PipeTransform {
+    transform(value: string | undefined): boolean|undefined {
+        return value ? value === 'true' : undefined;
```

Comment:
Using a null collease operator here may be useful, think in this context it's:

`return value === true ?? undefined`

Translates to "If `value` is true return true, otherwise return undefined`

Of course, test that, don't think I've used it within a conditional before

Reply:
I changed that around and it appears to work fine 👍